src/sdk/model/vdc.ts

```diff
@@ -263,4 +266,50 @@ export class Vdc extends Entity {
     });
   }
 
+  /**
+   * Gets the vDC metadata.
+   * @returns {Promise<Metadata<MetadataType>[]>}
+   */
+  async getMetadata(): Promise<Array<Metadata<MetadataType>>> {
+    return Iland.getHttp().get(`/vdcs/${this.uuid}/metadata`).then((response) => {
+      const jsonMetadata = response.data.data as Array<MetadataJson<MetadataType>>;
+      return jsonMetadata.map<Metadata<MetadataType>>((json) => {
+        switch (json.type) {
+          case 'number':
+            return new Metadata<number>(json as MetadataJson<number>);
+          case 'boolean':
+            return new Metadata<boolean>(json as MetadataJson<boolean>);
+          case 'datetime':
+            return new Metadata<Date>(json as MetadataJson<Date>);
+          case 'string':
+            return new Metadata<string>(json as MetadataJson<string>);
+        }
+        throw new Error(`Metadata with type ${json.type} is unknown.`);
+      });
+    });
+  }
+
+  /**
+   * Updates the vDC metadata.
+   * @param {Array<MetadataJson<MetadataType>>} metadata the new array of metadata
+   * @returns {Promise<Task>} task promise
+   */
+  async updateMetadata(metadata: Array<MetadataJson<MetadataType>>): Promise<Task> {
```

Comment:
shouldn't be using a Json interface as part of a method definition at this level